parser_test.go

```diff
@@ -235,7 +235,7 @@ func TestParser_ParseStatement(t *testing.T) {
 		})
 
 		// No column type
-		AssertParseStatement(t, `CREATE TABLE tbl (col1, col2)`, &sql.CreateTableStatement{
+		AssertParseStatement(t, `CREATE TABLE tbl (col1 INT, col2 INT)`, &sql.CreateTableStatement{
```

Comment:
While the SQL standard may require a column type, SQLite does not. See https://www.sqlite.org/lang_createtable.html

So this change needs to be reverted.

Reply:
From the README this SQL parser is targeted at SQLite syntax, in case you didn't notice.

Reply:
Sorry, I wasn't aware that SQLite allows column definitions with no data types. I've created a new PR to replace this one that eliminates the panic by doing a nil check instead of requiring a data type.